wrf_ens_tools/calc/calc.py

```diff
@@ -575,6 +575,42 @@ def dist_mask(xind, yind, xpts, ypts, r):
     """
     return (np.sqrt(((xind - xpts)**2) + ((yind - ypts)**2)) <= r)
 
+def ens_frequency(ens_field, thresh, field='counts'):
+    """
+    Calculate ensemble frequency of a given 2-dimensional variable for a specified threshold.
+
+    The number of ensemble members is inferred by the first dimension of `ens_field`.
+
```

Comment:
Add an axis argument to specify which axis is the ensemble members. Default can be axis=0